packages/@aws-cdk/aws-ecs/lib/base/task-definition.ts

```diff
@@ -428,6 +428,14 @@ export class TaskDefinition extends TaskDefinitionBase {
   private findContainer(containerName: string): ContainerDefinition | undefined {
     return this.containers.find(c => c.containerName === containerName);
   }
+
+  private isEphemeralPortMappingSupported(networkMode: NetworkMode): boolean {
+    return networkMode === NetworkMode.BRIDGE || networkMode === NetworkMode.NAT;
```

Comment:
nit: personally, I'd prefer these conditions in the if statement itself, just to allow for ease of debugging.

Reply:
No problem, will address that